beekeeper.yml

```diff
@@ -28,12 +22,18 @@ pull_request:
                 environment:
                     TEST_DIR: src/core
                     SRC_DEPENDANTS: src/dummy
+            - py3.7:
+                name: Python 3.7 Core API tests
+                task: py37
```

Comment:
Because I had to go look it up, this now exists in https://github.com/pybee/comb/pull/5

Reply:
@phildini Thanks for reviewing my PR, and merging toga/comb#5, I really appreciate it! I think we need @freakboy3742 (or someone with access) to add the py37 task to AWS before we merge this so that we don't break CI.

Reply:
FTR: I've uploaded the 3.7 comb image to Beekeeper.